parquet/pqarrow/schema.go

```diff
@@ -859,8 +867,10 @@ func mapToSchemaField(n *schema.GroupNode, currentLevels file.LevelInfo, ctx *sc
 func variantToSchemaField(n *schema.GroupNode, currentLevels file.LevelInfo, ctx *schemaTree, _, out *SchemaField) error {
 	// this is for unshredded variants. shredded variants may have more fields
 	// TODO: implement support for shredded variants
```

Comment:
Is this comment still up to date?

Reply:
bah, missed one! I can remove that now :) thanks!